source/real_locations.rst

```diff
@@ -0,0 +1,116 @@
+Checking the people in a location (SIR)
+=======================================
+
+What we were doing
+------------------
+
+While calculating the number of people who could potentially infect a person, we did the following:
+
+.. code-block:: scala
+
+    def computeInfectedFraction(node: Node, placeType: String, context: Context): Double = {
+    val totalNeighbourCount = node.getConnectionCount(node.getRelation[Person]().get)
+    if (totalNeighbourCount == 0)
+      return 0d
+    val infectedNeighbourCount = node.getConnectionCount(node.getRelation[Person]().get,
+      "infectionState" equ Infected).toDouble
+    infectedNeighbourCount / totalNeighbourCount.toDouble
+  }
+
+Let's take a closer look at the first line, how we calculate ``totalNeighbourCount``.
+
+.. code-block:: scala
+
+  val totalNeighbourCount = node.getConnectionCount(node.getRelation[Person]().get)
+
+Assume that the node was an ``Office``. In that case,
+
+.. code-block:: scala
+
+  node.getRelation[Person]().get
+
+returns the ``EMPLOYER_OF`` string. Now, ``totalNeighbourCount`` is counting the total number of nodes which the workplace node is the ``EMPLOYER_OF``.
```

Comment:
Grammar and explanation could be a little better here. I would prefer it if you said something like:

> Assume that the node in question is an ``Office``. In _this_ case [...]

and 

> Therefore, ``totalNeighbourCount`` counts the total number of nodes linked to this particular ``Office`` node with the ``EMPLOYER_OF`` relation.